### PR TITLE
Remove old stuff

### LINKED ISSUE #1631: Remove unused types, functions, cache keys; Remove preoptimized cache behavior
- Remove unused types
- Remove unused cache keys
- Remove stale functions
- Remove preoptimized cache behavior

### PR DESCRIPTION
**Pull request by adamgall**
Closes #1631 

- Remove an unused type.
- Remove an unused cache key.
- Remove a function which returns if a given network supports ENS or not.
- Remove ENS name resolution caching.

See my review comments for more context.

### CODE REVIEW

File: src/types/daoProposal.ts [hunk 1]
```diff
@@ -33,10 +33,6 @@ export interface AzoriusProposal extends GovernanceActivity {
   startBlock: bigint;
 }
 
-export interface AzoriusERC721Proposal extends AzoriusProposal {
-  votes: ERC721ProposalVote[];
-}
-
```
Comment (adamgall) [inline src/types/daoProposal.ts:39]: This wasn't being used anywhere.

File: src/hooks/utils/cache/cacheDefaults.ts [hunk 2]
```diff
@@ -42,7 +40,6 @@ interface IndexedObject {
  */
 export const CACHE_DEFAULTS: IndexedObject = {
   [CacheKeys.FAVORITES.toString()]: Array<string>(),
-  [CacheKeys.AUDIT_WARNING_SHOWN.toString()]: true,
```
Comment (adamgall) [inline src/hooks/utils/cache/cacheDefaults.ts:45]: `AUDIT_WARNING_SHOWN` not used anywhere

File: src/hooks/utils/cache/cacheDefaults.ts [hunk 3]
```diff
@@ -24,8 +24,6 @@ export enum CacheExpiry {
  */
 export enum CacheKeys {
   FAVORITES = 'favorites',
-  AUDIT_WARNING_SHOWN = 'audit_warning_shown',
```
Comment (adamgall) [inline src/hooks/utils/cache/cacheDefaults.ts:27]: `AUDIT_WARNING_SHOWN` not used anywhere

File: src/helpers/crypto.ts [hunk 4]
```diff
@@ -189,7 +188,3 @@ export const encodeMultiSend = (txs: MetaTransaction[]): string => {
 export function getEventRPC<T>(connection: ContractConnection<T>): T {
   return connection.asProvider;
 }
-
-export function supportsENS(chainId: number): boolean {
-  return chainId === sepolia.id || chainId == mainnet.id || chainId == sepolia.id;
-}
```
Comment (adamgall) [inline src/helpers/crypto.ts:195]: Let wagmi handle this

File: src/hooks/utils/useAddress.ts [hunk 5]
```diff
@@ -42,11 +36,6 @@ const useAddress = (addressInput: string | undefined) => {
       return;
     }
 
-    // only continue with ENS checks if the chain actually supports ENS
-    if (!supportsENS(chain.id)) {
-      return;
-    }
-
```
Comment (adamgall) [inline src/hooks/utils/useAddress.ts:49]: Let the provider handle this.

File: src/hooks/utils/cache/cacheDefaults.ts [hunk 6]
```diff
@@ -24,8 +24,6 @@ export enum CacheExpiry {
...
-  ENS_RESOLVE_PREFIX = 'ens_resolve_', // name.eth -> 0x0 caching
```
Comment (adamgall) [inline src/hooks/utils/cache/cacheDefaults.ts:28]: Quite simply, I don't think we should be caching ENS name resolutions. Names can change which address they're resolved to, and this doesn't seem like the kind of thing that we need to be worried about caching for performance sake.

**Review by Da-Colon — APPROVED** [review]
👍

**Pull request merged**